### ISSUE TITLE
Loss Function of ProGraML

### ISSUE THREAD

**Issue by ganler** [ChrisCummins/ProGraML#186]
Thanks for the great work! Just curious about the loss functions used for dataflow analysis and the downstream tasks.

**Comment by ChrisCummins** [comment]
Hi @ganler, we used [binary cross entropy loss](https://github.com/ChrisCummins/ProGraML/blob/a8fff63f0763e908b233dafdbc0ff9182e7a895a/models/ggnn/loss.py#L21-L62).

Cheers,
Chris
